wallet/walletrpc.c

```diff
@@ -657,6 +657,8 @@ static struct command_result *json_listfunds(struct command *cmd,
 						   "amount_msat");
 			json_add_txid(response, "funding_txid",
 				      &c->funding_txid);
+			json_add_num(response, "funding_output",
```

Comment:
nit: `funding_outnum` ?

Reply:
It already uses `txid` and `output` (for `outnum`) in `outputs`, so I think it's better to have consitency here with `channels` too.